### PR TITLE
ham/eng 316 rainbow wagmi code cleanup

### PR DESCRIPTION
**Pull request by ham-evans**
- Deleting old wallet stuff
- code cleanup

### CODE REVIEW

**Comment by illepic** [comment]
@ham-evans hit the conflicts and I'll review!

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# REDWOOD FROM HERE ON
+DATABASE_URL="postgresql://hamevans@localhost:5432/ikdb_dev?connection_limit=1"
```
Comment (chair28980) [inline .env]: @ham-evans I don't think we're committing the `.env` file to github

**Review by chair28980 — CHANGES_REQUESTED** [review]
Remove .env file

**Review by chair28980 — DISMISSED** [review]
i think these files shouldnt be merge either

.yarn/install-state.gz
.redwood/telemetry.txt

**Review by chair28980 — APPROVED** [review]

**Pull request merged**